common.h

```diff
@@ -16,6 +16,12 @@
 #define __packed __attribute__((__packed__))
 #endif /* __packed */
 
+#if __has_attribute(__fallthrough__)
```

Comment:
We may want to do the same thing as `libnvme` and have `fallthrough` configured be meson and defined in `config.h`.

Ref: https://github.com/linux-nvme/libnvme/blob/master/meson.build#L234

Reply:
Thanks for your reviewing and comments. Just fixed the patch as mentioned and rebased the changes with the latest master branch.